restler/restler.py

```diff
@@ -312,15 +315,18 @@ def signal_handler(sig, frame):
         settings = restler_settings.RestlerSettings(user_args)
     except restler_settings.InvalidValueError as error:
         print(f"\nArgument Error::\n\t{error!s}")
+        logger.exception(f"Argument error with {user_args}")
         sys.exit(-1)
     except Exception as error:
         print(f"\nFailed to parse user settings file: {error!s}")
+        logger.exception(f"Argument error with {user_args} failed to parse settings file.")
```

Comment:
>Argument error with {user_args} f [](http://example.com/codeflow?start=27&length=33)

Could you please change this to 'Failed to parse settings file with {user_args}'?